src/test/java/api/exercise/Exercise1.java

```diff
@@ -0,0 +1,108 @@
+package api.exercise;
+
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused", "MismatchedQueryAndUpdateOfCollection"})
+public class Exercise1 {
+
+    enum Status {
+        UNKNOWN,
+        PENDING,
+        COMMENTED,
+        ACCEPTED,
+        DECLINED
+    }
+
+    @Test
+    public void acceptGreaterThan21OthersDecline() {
+        Person alex = new Person("Алексей", "Мельников", 20);
+        Person ivan = new Person("Иван", "Стрельцов", 24);
+        Person helen = new Person("Елена", "Рощина", 22);
+        Map<Person, Status> candidates = new HashMap<>();
+        candidates.put(alex, Status.PENDING);
+        candidates.put(ivan, Status.PENDING);
+        candidates.put(helen, Status.PENDING);
+
+        // TODO реализация
+        candidates.replaceAll(((person, status) ->
+                person.getAge() > 21 ? Status.ACCEPTED : Status.DECLINED));
+
+        assertEquals(Status.ACCEPTED, candidates.get(ivan));
+        assertEquals(Status.ACCEPTED, candidates.get(helen));
+        assertEquals(Status.DECLINED, candidates.get(alex));
+    }
+
+    @Test
+    public void acceptGreaterThan21OthersRemove() {
+        Person alex = new Person("Алексей", "Мельников", 20);
+        Person ivan = new Person("Иван", "Стрельцов", 24);
+        Person helen = new Person("Елена", "Рощина", 22);
+        Map<Person, Status> candidates = new HashMap<>();
+        candidates.put(alex, Status.PENDING);
+        candidates.put(ivan, Status.PENDING);
+        candidates.put(helen, Status.PENDING);
+
+        candidates.put(new Person("a", "a", 19), Status.PENDING);
+        candidates.put(new Person("b", "c", 16), Status.PENDING);
+        candidates.put(new Person("b", "c", 5), Status.PENDING);
+
+        // TODO реализация
+        candidates.keySet()
+                  .removeIf(person -> person.getAge() <= 21);
+        candidates.replaceAll(
+                ((person, status) -> person.getAge() > 21 ? Status.ACCEPTED : status));
```

Comment:
It's OK, but no need for comparison because at 57 line you have removed all "otherwise" elements.